libs/transactions/src/lib/views/transactions-view/transactions-view.component.html

```diff
@@ -9,9 +9,18 @@ <h1 *ngIf="title !== ''" data-role="transactions-view__title">
       </div>
       <ul *ngIf="transactions$ | async as items; else loading" class="bb-list">
         <li
-          *ngFor="let item of items | filterTransactions: filter"
+          *ngFor="
+            let item of items | filterTransactions: filter:selectedAccount
```

Comment:
Isn't better to combine `transactions$` and `queryParams` Observables and do filtering in rxjs pipe?

Reply:
Done